mozilla-release/browser/confvars.sh

```diff
@@ -35,8 +35,8 @@ MOZ_SERVICES_COMMON=1
 MOZ_SERVICES_CRYPTO=1
 MOZ_SERVICES_HEALTHREPORT=1
 MOZ_SERVICES_METRICS=1
-MOZ_SERVICES_SYNC=1
```

Comment:
BTW, we may also want to disable mozilla health report service (see line 36). @luciancor ?